index.html

```diff
@@ -0,0 +1,161 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>CC Global Fashion Summit</title>
+  <link rel="stylesheet" href="stylesheets/home-style.css">
+  <script src="https://kit.fontawesome.com/eb94992e2c.js" crossorigin="anonymous"></script>
+  <script src="js/mobile-popup.js" defer></script>
+  <script src="js/speakers.js" defer></script>
+</head>
+<body>
+  <div class="wrapper">
+    <header>
+      <nav class="top-nav desktop-only">
+        <i class="fa-brands fa-facebook-f"></i>
+        <i class="fa-brands fa-twitter"></i>
+        <span class="color-3 font-2">English</span><span class="color-3 font-2">My Page</span><span class="color-3 font-2">Logout</span>
+      </nav>
+      <nav class="page-navigation">
+        <!-- mobile menu navigation overlay -->
+        <div id="mobile-menu-navigation" class="overlay mobile-only">
+          <a href="javascript:void(0)" class="close-btn">
+            <i class="fa-solid fa-xmark"></i>
+          </a>
+          <ul class="overlay-content font-2">
+            <li class="page-link">
+              <a href="pages/about.html">About</a>
+            </li>
+            <li class="page-link">
+              <a href="javascript:void(0)">Sponsor</a>
+            </li>
+            <li class="page-link">
+              <a href="javascript:void(0)">Join</a>
+            </li>
+            <li class="page-link">
+              <a href="javascript:void(0)">Programs</a>
+            </li>
+            <li class="page-link">
+              <a href="javascript:void(0)">News</a>
+            </li>
+          </ul>
+        </div>
+        <!-- end of mobile navigation overlay -->
+        <span class="ham-container color-2 mobile-only"><img class="hamburger" src="images/hamburger.png" alt="Hamburger"></span>
+        <div class="desktop-nav desktop-only">
+          <span id="cc-Logo">
+            <a class="logo" href="index.html">
+              <img src="images/CC-logo.svg" alt="Logo">
+            </a>
+          </span>
+          <ul class="desktop-nav-items color-2 font-2">
+            <li>
+              <a href="pages/about.html">About</a>
+            </li>
+            <li>Program</li>
+            <li>Sponsor</li>
+            <li>Join</li>
+            <li>News</li>
+          </ul>
+          <span class="logo-2 color-2 font-2">CC Campaign</span>
+        </div>
+      </nav>
+      <section class="header-content">
+        <p class="font-2 color-2 greeting">"Let's Talk Fashion Without Borders"</p>
+        <h1 class="font-1 hero color-1">CULTURE CUSTODIAN<br>GLOBAL FASHION SUMMIT<br>2022</h1>
+        <p class="title font-2 color-2">
+          The joyful event that explores the possibilities and impact of tech in the fashion industry, with guests from all over
+          the world, would take place in July, in Nigeria.
+        </p>
```

Comment:
### OPTIONAL
- [ ] Kindly adjust the width of the "title" paragraph and also align the background image by it to look similar to the  original design, this will make all texts in that section look aligned to the left

Reply:
Hi, just for more clarification. Is this for the mobile or desktop view?

Reply:
> Hi @prolajumokeoni, just for more clarification. Is this for the mobile or desktop view?

Desktop view @deyemiobaa